subo/release/check.go

```diff
@@ -14,67 +16,130 @@ import (
 	"github.com/suborbital/subo/subo/util"
 )
 
-func getTimestampCache() (bool, error) {
-	cachePath, err := util.GetCacheDir()
+func getTimestampCache() (time.Time, error) {
+	cachePath, err := util.CacheDir()
 	if err != nil {
-		return false, errors.Wrap(err, "failed to GetCacheDir")
+		return time.Time{}, errors.Wrap(err, "failed to CacheDir")
 	}
 
-	var cached_timestamp time.Time
-	filePath := filepath.Join(cachePath, "latest_version_check.txt")
-	_, err = os.Stat(filePath)
-	if os.IsNotExist(err) {
-		cached_timestamp = time.Time{}
+	cachedTimestamp := time.Time{}
+	filePath := filepath.Join(cachePath, "subo_last_checked.txt")
+	if _, err = os.Stat(filePath); os.IsNotExist(err) {
 	} else if err != nil {
-		return false, errors.Wrap(err, "failed to Stat")
+		return time.Time{}, errors.Wrap(err, "failed to Stat")
 	} else {
 		data, err := ioutil.ReadFile(filePath)
 		if err != nil {
-			return false, errors.Wrap(err, "failed to ReadFile")
+			return time.Time{}, errors.Wrap(err, "failed to ReadFile")
 		}
 
-		cached_timestamp, err = time.Parse(time.RFC3339, string(data))
+		cachedTimestamp, err = time.Parse(time.RFC3339, string(data))
 		if err != nil {
-			return false, errors.Wrap(err, "failed to parse cached timestamp")
+			return time.Time{}, errors.Wrap(err, "failed to parse cached timestamp")
 		}
 	}
+	return cachedTimestamp, nil
+}
 
-	// check if 1 hour has passed since the last version check, and update the cached timestamp if so
-	current_timestamp := time.Now().UTC()
-	if cached_timestamp.IsZero() || current_timestamp.After(cached_timestamp.Add(time.Duration(1)*time.Hour)) {
-		data := []byte(current_timestamp.Format(time.RFC3339))
-		if err := ioutil.WriteFile(filePath, data, os.ModePerm); err != nil {
-			return false, errors.Wrap(err, "failed to WriteFile")
+func cacheTimestamp(timestamp time.Time) error {
+	cachePath, err := util.CacheDir()
+	if err != nil {
+		return errors.Wrap(err, "failed to CacheDir")
+	}
+
+	filePath := filepath.Join(cachePath, "subo_last_checked.txt")
```

Comment:
```suggestion
	filePath := filepath.Join(cachePath, lastCheckedFilename)
```

So we don't have the same hardcoded string in two places.